src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/newQuestionForm.tag

```diff
@@ -28,21 +27,21 @@
                     <ul id="add-new-question-dropdown" class="dropdown-menu">
                         ${fqForm.questionTypeOptions}
                     </ul>
-
-                    <a href="/instructorHelp.jsp#fbQuestionTypes"
-                        target="_blank" rel="noopener noreferrer"> <i
-                        class="glyphicon glyphicon-info-sign"></i>
-                    </a> <a id="button_copy" class="btn btn-primary margin-bottom-7px" 
-                            data-actionlink="${data.instructorQuestionCopyPageLink}"
-                            data-fsname="${fqForm.feedbackSessionName}" data-courseid="${fqForm.courseId}"
-                            data-target="#copyModal" data-toggle="modal">
-                        Copy Question
-                    </a>
-                    <a id="button_done_editing" class="btn btn-primary margin-bottom-7px"
-                        href="${fqForm.doneEditingLink}">
-                        Done Editing
-                    </a>
                 </div>
+
+                <a href="/instructorHelp.jsp#fbQuestionTypes"
+                    target="_blank" rel="noopener noreferrer"> <i
+                    class="glyphicon glyphicon-info-sign"></i>
+                </a> <a id="button_copy" class="btn btn-primary margin-bottom-7px" 
```

Comment:
Since these lines are touched, let's do some cleanup here. The formatting can be improved as follows:
```html
<a href...
        ...>
    <i class=...></i>
</a>
<a id="button_copy" ...
```